doc/quickstart/grass_quickstart.rst

```diff
@@ -96,8 +93,7 @@ the `Map Display` canvas. When done go back to the Profile window.
 Click on the I/O button of the far right to close the profile window.
 
 Create a random surface
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
+================================================================================
 .. HB comment: this quickstart is getting kinda long, maybe retire this section.
```

Comment:
I agree, this is a bit long. Perhaps this section could be retired?

Reply:
ok for me

Reply:
okay, I removed it.